README.md

```diff
@@ -20,23 +20,35 @@ with an explanation if you would like access to this for some reason. Otherwise,
 replace the `spring.datasource` variables in `application.properties` with your
 own database to proceed. 
 
-Run `./mvnw clean install` to build.
+* Run `./mvnw clean install` if it's your first time running the app (or for whatever
+reason you need to re-download Maven dependencies). 
 
-Run `./mvnw test` to run tests.
+* Run `./mvnw test` to run the project's unit tests.
```

Comment:
It's not just the unit tests, right? Doesn't this run the API / integration tests as well?

Reply:
Yeah - I'll update this to say "and API tests"